compile in suggester fails due to missing checkstyle/fileheader.txt

When `mvn compile` is run in the `suggester` directory, it fails with:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check (checkstyle) on project suggester: Failed during checkstyle configuration: cannot initialize module Header - illegal value 'checkstyle/fileheader.txt' for property 'headerFile' of module Header: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to find: checkstyle/fileheader.txt -> [Help 1]
```

--- a/checkstyle/style.xml
+++ b/checkstyle/style.xml
@@ -5,7 +5,7 @@
 <module name="Checker">
     <module name="NewlineAtEndOfFile"/>
     <module name="Header">
-        <property name="headerFile" value="checkstyle/fileheader.txt"/>
+        <property name="headerFile" value="${checkstyle.header.file}"/>
     </module>
     <module name="FileLength"/>
     <module name="FileTabCharacter"/>
@@ -30,8 +30,5 @@
             <property name="offCommentFormat" value="&lt;editor-fold defaultstate=&quot;collapsed&quot; desc=&quot;Generated Code&quot;&gt;"/>
             <property name="onCommentFormat" value="&lt;/editor-fold&gt;"/>
         </module>
-    </module>
-    <module name="SuppressionFilter">
-        <property name="file" value="checkstyle/suppressions.xml" />
     </module>
 </module>

--- a/pom.xml
+++ b/pom.xml
@@ -225,6 +225,39 @@ Portions Copyright (c) 2018, Chris Fraire <cfraire@me.com>.
               <version>2.9</version>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.0.0</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.puppycrawl.tools</groupId>
+                            <artifactId>checkstyle</artifactId>
+                            <version>8.11</version>
+                        </dependency>
+                    </dependencies>
+                    <executions>
+                        <execution>
+                            <id>checkstyle</id>
+                            <phase>validate</phase>
+                            <configuration>
+                                <configLocation>/checkstyle/style.xml</configLocation>
+                                <encoding>UTF-8</encoding>
+                                <consoleOutput>true</consoleOutput>
+                                <failsOnError>true</failsOnError>
+                                <suppressionsLocation>/checkstyle/suppressions.xml</suppressionsLocation>
+                                <headerLocation>/checkstyle/fileheader.txt</headerLocation>
+                            </configuration>
+                            <goals>
+                                <goal>check</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
     </build>
 
     <reporting>

--- a/suggester/pom.xml
+++ b/suggester/pom.xml
@@ -92,29 +92,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>3.0.0</version>
-                <dependencies>
-                    <dependency>
-                        <groupId>com.puppycrawl.tools</groupId>
-                        <artifactId>checkstyle</artifactId>
-                        <version>8.11</version>
-                    </dependency>
-                </dependencies>
-                <executions>
-                    <execution>
-                        <id>checkstyle</id>
-                        <phase>validate</phase>
-                        <configuration>
-                            <configLocation>../checkstyle/style.xml</configLocation>
-                            <encoding>UTF-8</encoding>
-                            <consoleOutput>true</consoleOutput>
-                            <failsOnError>true</failsOnError>
-                        </configuration>
-                        <goals>
-                            <goal>check</goal>
-                        </goals>
-                    </execution>
-                </executions>
             </plugin>
 
             <plugin>

